lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3565,11 +3567,43 @@ def test_exam_no_redirect(self):
         self.section2.is_time_limited = True
         self.store.update_item(self.section2, self.user.id)
 
-        lms_url, mfe_url = self._get_urls()  # lint-amnesty, pylint: disable=unused-variable
+        lms_url, __, __ = self._get_urls()
 
         assert self.client.get(lms_url).status_code == 200
 
 
+@ddt.ddt
+class PreviewRedirectTests(BaseViewsTestCase):  # lint-amnesty, pylint: disable=missing-class-docstring
```

Comment:
Thanks for adding these tests!

Consider listening to pylint here, and adding a class docstring.

Btw, if you're going to disable pylint warnings, you don't want include the `lint-amnesty` part. That is a tag to indicate that the pylint-disabling was part of a one-time mass-pylint-warning-forgiveness effort we did recently. Just `pylint: disable=warning-name-here` will do the trick.